src/main/resources/application.yaml

```diff
@@ -58,14 +58,14 @@ spring:
     username: ${DB_USERNAME:folio_admin}
   jpa:
     open-in-view: true
-    database-platform: org.hibernate.dialect.PostgreSQL10Dialect
+    database-platform: org.folio.notes.config.hibernate.ExtendedPostgreSQL10Dialect
     hibernate:
       ddl-auto: none
     properties:
       hibernate:
-        dialect: org.hibernate.dialect.PostgreSQL10Dialect
+        dialect: org.folio.notes.config.hibernate.ExtendedPostgreSQL10Dialect
         format_sql: true
-    show-sql: false
+    show-sql: true
```

Comment:
I suppose this parameter should be set to `true` only for debug purposes

Reply:
You are right.